.github/workflows/radar-snapshot-action.yml

```diff
@@ -0,0 +1,33 @@
+# Trigger an SDK publication to SNAPSHOTS repo on Maven Central when a commit is pushed to develop.
+name: Radar Snapshot Action
+on:
+  push:
+    branches:
+      - develop
+jobs:
+  Build-And-Publish:
+    runs-on: ubuntu-latest
+    steps:
+      # Prepare Environment
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Configure Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
+      # Gradle Build & Publish
+      - name: Build SDK
+        run: ./gradlew build
+      - name: Publish to Maven #and dispatch snapshot event
+        env:
+          NEXUS_USERNAME: ${{ secrets.NEXUS_USERNAME }}
+          NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+          ORG_GRADLE_PROJECT_SIGNINGKEY: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGKEY }}
+          ORG_GRADLE_PROJECT_SIGNINGPASSWORD: ${{ secrets.ORG_GRADLE_PROJECT_SIGNINGPASSWORD }}
+#          PAT_USERNAME: ${{ secrets.PAT_USERNAME }}
+#          PAT_TOKEN: ${{ secrets.PAT_TOKEN }}
```

Comment:
are these env vars needed or nah?

Reply:
These are needed, but we do not have them in GitHub yet.